src/main/java/uk/nhs/hee/tis/revalidation/core/service/AdminService.java

```diff
@@ -60,9 +62,34 @@
 
     // TODO: A limited number of users can be returned before pagination occurs, handle pagination.
     ListUsersInGroupResult listUsersResult = identityProvider.listUsersInGroup(request);
+    List<UserType> userTypeList = listUsersResult.getUsers();
 
-    return listUsersResult.getUsers().stream()
-        .map(mapper::toDto)
-        .collect(Collectors.toList());
+    return buildAdminDtoList(userTypeList);
+
+  }
+
+  private List<AdminDto> buildAdminDtoList(List<UserType> userTypeList) {
+    List<AdminDto> adminDtoList = new ArrayList<>();
+    userTypeList.stream().forEach(userType -> {
+      AdminDto adminDto = new AdminDto();
+      adminDto.setUsername(userType.getUsername());
+      adminDto.setFullName(getUserFullName(userType.getAttributes()));
+      adminDtoList.add(adminDto);
```

Comment:
Please use .Collect() method instead of manually adding into list.

Reply:
@uzairbaig I have upadated the code. Thanks